egs/libri_css/asr1_mono/run.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# Copyright  2020  University of Stuttgart (Author: Pavel Denisov)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=-1       # start from -1 if you need to start from data download
+stop_stage=100
+nj=32
+
+# feature configuration
+dumpdir=dump   # directory to dump full features
+do_delta=false
+
+# decoding parameter
+recog_model=model.val5.avg.best  # set a model to be used for decoding
```

Comment:
Maybe it's better to add some comments that they are trained from https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/RESULTS.md#pytorch-large-transformer-with-specaug-4-gpus--transformer-lm-4-gpus

Reply:
Thanks @akreal. @sw005320 Looks good to me.

Reply:
Added.